src/hotspot/share/runtime/vmThread.hpp

```diff
@@ -39,15 +39,15 @@ class VMOperationTimeoutTask : public PeriodicTask {
 private:
   volatile int _armed;
   jlong _arm_time;
-
+  const char* _vm_op_name = nullptr;
```

Comment:
Style question: are we doing these inits inline, instead of constructor?

Reply:
My mistake; I will fix it.